jenkinsapi_tests/unittests/test_crumbed_requester.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
+import requests
```

Comment:
Not used here.

Please make systest instead of unittest - it is always better to test using real Jenkins instance and there is no need to monkeypatch anything.